### ISSUE TITLE
Server-side rendering object yields [object Object]

### ISSUE THREAD

**Issue by graforlock** [shuhei/pelo#9]
### Expected behavior
When I pass input like this:

```javascript
<a href="/link" ${title ? {title} : '' }>Link</a>
```

the rendered markup is 

```javascript
<a href="/link" title="My title" >Link</a>
```
on the server.

### Actual behavior
When I pass input like this:

```javascript
<a href="/link" ${title ? {title} : '' }>Link</a>
```

the rendered markup is 

```javascript
<a href="/link" [object Object] >Link</a>
```
on the server.

### Steps to reproduce behavior
Try to render markup  on server side.

Sometimes for SEO purposes, you'd want to avoid leaving blank attributes like `<a href="/link" title>Link</a>`, hence the need to have a good way of not rendering these at all.

Passing object is fine on the client-side with `bel` / `hyperx`, but it has different behaviour on the server with `pelo`.

As per original issue at [choo 591](https://github.com/choojs/choo/issues/591) (where it was caught)

**Comment by shuhei** [comment]
@graforlock  Published 0.0.4 with your pull requests. Thanks for your awesome work!

**Comment by graforlock** [comment]
Cool! no problem @shuhei

**Comment by lorengreenfield** [comment]
Hey guys, just an FYI, this addition breaks csjs

**Comment by shuhei** [comment]
@lorengreenfield Thanks for letting is know! Do you mind giving us an example or elaborating a bit? I have no clue with csjs.

**Comment by graforlock** [comment]
@lorengreenfield 

`csjs` does look like a very non-standard way of using `pelo` with all this funky experimental css modules stuff. `pelo` is designed to work with libs like `yoyo` or `choo` to render DOM/HTML server-side. You guys can lock it to 0.0.3 or submit a patch I guess to meet both needs. 

You might also fork it since the use case is going quite the opposite direction vs `pelo`'s main use case.

The desire of the PR was to replicate the same missing functionality and reaching parity that aforementioned libraries like `bel` have for the same input:

`<div ${{ 'data-src': '123434' }} >Text</div>` 

It would work fine with `bel` but yield `[object Object]` with `pelo`, and this definitely should work the same for both since  otherwise it has vastly inconsistent  server side output vs client-side.

**Comment by lorengreenfield** [comment]
@graforlock @shuhei - I did think of forking or locking down to 0.0.3 might be a good approach, but I think you guys have done the right thing advancing pelo to match bel. It'd be up to csjs to change to match pelo 0.0.4. 

At the moment, csjs produces an object as a value, but also adds a `toString` method which returns a scoped css class as a string, so it's correctly returning a string when run through pelo, instead of [object Object].

I've managed to make a work around for my needs by forcibly running the .toString method if it exists, like so:

```
let html = (strings, ...values) => {
  values = values.map(value => {
    if (value && value.hasOwnProperty('toString')) {
      return value.toString()
    }
    return value
  })

  return bel(strings, ...values)
}
```
This way I can also start using the feature you've just added.

Keep up the great work! I really appreciate the time that's gone into pelo and all the other hyperX related libraries, so thanks.
